tests/test_rxnav/rx_response.json

```
@@ -0,0 +1 @@
+{"idGroup":{"idType":"NDC","id":"0023-6012","rxnormId":["892598"]}}
```

Pretty printing please :)
```suggestion
{
  "idGroup": {
    "idType": "NDC",
    "id": "0023-6012",
    "rxnormId": [
      "892598"
    ]
  }
}
```